docs/cs-manual/source/record-types.rst

```diff
@@ -0,0 +1,64 @@
+Record Types
+############
+
+A game world contains many items, such as chests, weapons and monsters. All
+these items are merely instances of templates we call *Objects*. The OpenMW CS
+*Objects* table contains information about each of these template objects, eg.
+its value and weight in the case of items, or an aggression level in the case
+of NPCs.
+
+Let’s go through all Record Types and discuss what you can tell OpenMW CS about
+each of them.
+
+Activator
+   When the player enters the cell containing this object a script is started.
+   Often it also has a Script attached to it, though it is not mandatory. These
```

Comment:
This is a bit confusing. Makes it sounds as if there were two scripts involved. What actually happens is that the script attached to the activator (not mandatory indeed) will run once per frame as long as the cell the activator is in is active.